icu4c/source/common/dictbe.cpp

```diff
@@ -1342,11 +1376,22 @@ CjkBreakEngine::divideUpDictionaryRange( UText *inText,
         t_boundary.addElement(numCodePts, status);
         numBreaks++;
     } else {
```

Comment:
could we keep to original "} else {" block unchange but insert a } else if (isPhraseBreaking) {" block for the new code?